static/js/Autocomplete.jsx

```diff
@@ -244,7 +288,8 @@ const SearchInputBox = ({getInputProps, suggestions, highlightedIndex,
       </div>
     );
   };
-const SuggestionsDispatcher = ({ suggestions, getItemProps, highlightedIndex}) => {
+const SuggestionsDispatcher = ({ suggestions, getItemProps, highlightedIndex,
+                                            _submitSearch, _redirectToObject}) => {
```

Comment:
what does it mean to have an underscore here? normally that means private but it's not private if it's a prop.